api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -590,6 +590,10 @@ public int getTableRowCount(String tableName) {
 
   @Override
   public int getAppUsageRowCount(String tableName) {
+    if (!workbenchConfigProvider.get().reporting.exportTerraDataWarehouse) {
```

Comment:
running the Local reporting cron fails with NPE without this check, because the config values are not set below.